src/SDKs/ContainerRegistry/ContainerRegistry.Tests/ContainerRegistry.Tests.csproj

```diff
@@ -12,9 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="[5.0.2-preview,6.0)" />
-    
-    <!--<PackageReference Include="Microsoft.Azure.Management.ContainerRegistry" Version="1.2.0-preview" />-->
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="[7.0.0-preview,7.0)" />
```

Comment:
@djyou you should be using latest version of Resource Manager 1.6.0 at least or the latests version to record all your tests.
So sorry to remind you in my earlier iteration of code review.
You are still using 1.1.0 version which is very old and should not be used to create your resources.

Reply:
I was confused with the comment actually. Did you mean we should add reference to Resource Manager 1.6.0 because currently it's not in csproj file?